storybook/src/components/PasswordInput/PasswordInput.stories.tsx

```diff
@@ -13,11 +13,16 @@ const meta = {
   args: {
     disabled: false,
     invalid: false,
+    size: 0,
```

Comment:
This shows the associated code as `<PasswordInput size={0} />`. We should probably remove this initial value to prevent people from inadvertently pasting it into their applications. The downside is that they must press the ‘Set Number’ button first, but this is how Storybook distinghuishes undefined and zero values.

```suggestion
```